crates/sui-indexer/migrations/2023-06-28-174550_object_balances/up.sql

```diff
@@ -0,0 +1,11 @@
+-- Your SQL goes here
+
+CREATE TABLE object_balances
+(
+    id          address NOT NULL,
+    version     BIGINT  NOT NULL,
+    coin_type   VARCHAR NOT NULL,
+    balance     BIGINT  NOT NULL,
+
+    CONSTRAINT object_balances_pk PRIMARY KEY (id, version, coin_type)
```

Comment:
do we need to keep the whole history of coin balances? I thought we only keep the updated coin balances (latest version) on FN. cc @longbowlu

Reply:
This is slightly different from the coin balances index on FN in that this tracks all balances of any `Coin<T>` nested into all objects. That is, each object being tracked isn't a Coin type but could be something that has balances inside of it.